src/main.c

```diff
@@ -48,7 +48,7 @@ int main ()
 		// drawing
 		BeginDrawing();
 
-		// Setup the backbuffer for drawing (clear color and depth buffers)
+		// Setup the back-buffer for drawing (clear color and depth buffers)
```

Comment:
I have never seen it called a "back-buffer". "back buffer" seems to be the most common.